docs/src/tutorials/bseries_basics.ipynb

```diff
@@ -0,0 +1,653 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In this notebook we use `bseries.jl` to investigate error expansions for RK methods applied to specific problems."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Load the packages we will use.  These must first be installed using: import Pkg; Pkg.add(\"package_name\")\n",
+    "using BSeries\n",
+    "using Latexify\n",
```

Comment:
This is only required for pretty printing with `latexify` and not required for the other core functionality. It may be nice to clarify this.